packages/gen-ai/frontend/src/app/AIAssets/components/AIModelTableRow.tsx

```diff
@@ -0,0 +1,180 @@
+                          Resource type
+                        </Content>
+                      </FlexItem>
+                      <FlexItem>
+                        <Content style={{ color: 'var(--pf-t--global--text--color--subtle)' }}>
+                          InferenceService
+                        </Content>
+                      </FlexItem>
+                    </Flex>
+                  </FlexItem>
+                </Flex>
+              }
+            >
+              <Button
+                variant={ButtonVariant.plain}
+                aria-label={`Resource information for ${model.model_name}`}
+              >
+                <InfoCircleIcon />
+              </Button>
+            </Popover>
+          </div>
+        }
+        description={model.description}
+      />
+    </Td>
+    <Td dataLabel="Internal endpoint">
+      {model.internalEndpoint ? (
+        <Popover
+          position="right"
+          headerContent={
+            <div style={{ padding: '16px 16px 0 16px' }}>
+              <div style={{ margin: 0, fontSize: 'var(--pf-t--global--font--size--h3)' }}>
+                Internal Endpoint URL
+              </div>
+            </div>
+          }
+          bodyContent={
+            <div style={{ padding: '0 16px 16px 16px' }}>
+              <ClipboardCopy
+                hoverTip="Copy"
+                clickTip="Copied"
+                style={{ fontFamily: 'monospace' }}
+                aria-label={`Internal endpoint URL for ${model.model_name}`}
+              >
+                {model.internalEndpoint}
+              </ClipboardCopy>
+            </div>
+          }
+        >
+          <Button variant={ButtonVariant.link}>View</Button>
+        </Popover>
+      ) : (
+        '—'
+      )}
+    </Td>
+    <Td dataLabel="External endpoint">
+      {model.externalEndpoint ? (
+        <Popover
+          position="right"
+          headerContent={
+            <div style={{ padding: '16px 16px 0 16px' }}>
+              <div style={{ margin: 0, fontSize: 'var(--pf-t--global--font--size--h3)' }}>
+                External Endpoint URL
+              </div>
+            </div>
+          }
+          bodyContent={
+            <div style={{ padding: '0 16px 16px 16px' }}>
+              <ClipboardCopy
+                hoverTip="Copy"
+                clickTip="Copied"
+                style={{ fontFamily: 'monospace' }}
+                aria-label={`External endpoint URL for ${model.model_name}`}
+              >
+                {model.externalEndpoint}
+              </ClipboardCopy>
+            </div>
+          }
+        >
+          <Button variant={ButtonVariant.link}>View</Button>
+        </Popover>
+      ) : (
+        <Button
+          variant={ButtonVariant.link}
+          icon={<PlusCircleIcon color="var(--pf-t--global--palette--blue-600)" />}
+          isDisabled // TODO: Enable when external endpoint creation api is implemented
+        >
+          Create
+        </Button>
```

Comment:
From the discussion https://redhat-internal.slack.com/archives/C099MEPGF43/p1758209924990989?thread_ts=1758204451.054829&cid=C099MEPGF43, this will not be supported for 3.0. We can show something else instead of showing a disabled create button.